Cargo.toml

```diff
@@ -17,10 +17,10 @@ rustls-pemfile = "0.3.0"
 webpki = "0.22"
 webpki-roots = "0.22"
 ring = "0.16.5"
-untrusted = "0.7.0"
+untrusted = "0.7.1"
```

Comment:
Why did you upgrade this? This should be pulled in automatically, since it's semver-compatible.

Reply:
For pre 1.0 versions, a "minor" version bump is considered to be the same as a major bump post 1.0, is it not?

Reply:
This is not a minor bump, this is a micro bump (major = 0, minor = 7, micro = 1).

Reply:
Oh, my bad, I didn't review the comment correctly 🙈